package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -223,11 +222,6 @@ wireless.foreach_radio(uci, function(radio, index, config)
 		uci:set('wireless', radio_name, 'legacy_rates', false)
 		configure_mesh_wireless(radio, index, config)
 	elseif (band == '5g') then
-		-- ToDo: Remove in v2024.x
-		local hostapd_options = uci:get_list('wireless', radio_name, 'hostapd_options')
```

Comment:
This migration was added in 2023.1, I think it is too early to remove it.

Reply:
What do you mean?
v2023.1 means: The migration is for v2022.1 and v2021.1
from v2022.1 you can upgrade to v2023.2 but not till gluon main/v2025.1

Removing this should therefore be fine, right? (Even the code comment says so)

Reply:
@Djfe whats the problem with leaving it for another release?

Reply:
conclusion in the meeting was:
let's keep these lines (don't remove them) and then this PR will be merged :)
They don't harm.
We will remove them later

Reply:
@maurerle :)

Reply:
Done :) LGTM :)